packages/varnish/src/components/logos/AI2Projects.tsx

```diff
@@ -4,7 +4,7 @@ export interface IconProps {
   width?: string;
   height?: string;
   className?: string;
-  mono?: boolean;
+  polychrome?: boolean;
```

Comment:
i think you have a flag in a different file that says mono={true}
so.. is that cleaned up?

Reply:
yep sorry about that pushed too early.... fixing